src/components/course/data/service.js

```diff
@@ -38,6 +38,11 @@ export default class CourseService {
 
     const courseData = camelCaseObject(courseDataRaw);
     const courseDetails = courseData[0];
+
+    if (!courseDetails.advertisedCourseRunUuid) {
+      courseDetails.advertisedCourseRunUuid = courseDetails.courseRuns[0].uuid;
+    }
```

Comment:
This can be ignored; just a temporary hack because the E2E course does not have an `advertised_course_run_uuid` so I'm "hardcoding" it here.